conifer/model.py

```diff
@@ -114,12 +115,13 @@ def draw(self, filename : str = None, graph=None, tree_id=None):
     return graph
 
   def apply(self, X):
+    comp_func = lambda arg1, arg2: arg1 < arg2 if self.splitting_convention == '<' else arg1 <= arg2
```

Comment:
I'd like this to be defined in the class `__init__`, and again with a check on the validity on the string (e.g. defaulting to `<=` whenever the string is not `'<'` is not totally robust)